File: 1580-shuffle-the-array/1580-shuffle-the-array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    let res = Array(nums.length);
    let ind=0;
    for(let i=0;i<n;i++){
        res[ind++] = nums[i];
        res[ind++] = nums[i+n];
    }
    return res;
};

File: 1580-shuffle-the-array/1580-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int n2 = nums.length/2;int in=0;
        int[] res = new int[nums.length];
        for(int i=0;i<n;i++){
            res[in] = nums[i];
            in++;
            res[in] = nums[i+n];
            in++;
        }
        return res;
    }
}